assignments/week_1/exercise_1_2/exercise_1_2.md

```diff
@@ -0,0 +1,231 @@
+# Exercise 1.2
+Welcome to your second exercise of RoboJackets software training! This exercise will be testing your knowledge of basic C++ syntax and vector manipulation. For this exercise we expect you to have watched or know the content of the Codecademy C++ course up through vectors.
+
+# Exercise Objective
+In this exercise, you will be reversing an vector and checking if a string is a palindrome. The purpose of this exercise is to get you familar with working with vectors and strings. This exercise should help solidify your knowledge of loops, variables, conditionals, and vectors. _We aren't going to be using any new C++ standard library functions in the instructions, but feel free to use them if you want._
+
+# Starting Code
+We have already given you a main() function to write your code in and have included the proper libraries for you to use. We have also set up most of variable initilization and commented where you will write your code.
+
+
+### Part 1: Vector Reversal
+```c++
+#include <iostream>
+#include <vector>
+
+int main()
+{
+    // Part 1: Vector Reversal
+
+    // Input vector from command-line
+    std::vector<int> list;
+    int input;
```

Comment:
We should initialize all variables in our examples to encourage that habit in the new members.
```suggestion
    int input = 0;
```